docs/source/index.rst

```diff
@@ -0,0 +1,30 @@
+:github_url: https://github.com/carperai/cheese
```

Comment:
Great documentation, we should link to it to the readme.

Reply:
will add to readme in main :)